lib/rspec/rabl/attribute_matcher.rb

```diff
@@ -11,7 +11,7 @@ def description
       end
 
       def failure_message
-        "expected #{parsed} to render #{rendered_attribute} = #{expected_value}"
+        "expected #{expected_value.inspect} in #{attribute_path}\n  got #{rendered_value.inspect}"
```

Comment:
Actually, this is probably expected behavior. For example, if I give `DateTime.now`, I would expect to see a `DateTime` instance rendered, not the string. This is the current behavior, anyways.

```
Failure/Error: it{ expect(subject).to render_attribute(:guid).with_value(DateTime.now) }
  expected "#<DateTime: 2015-10-28T22:50:30-06:00 ((2457325j,17430s,396034000n),-21600s,2299161j)>" in ["users"]["user"]["guid"]
    got "abc"
```